src/hotspot/cpu/riscv/vm_version_riscv.cpp

```diff
@@ -464,3 +465,18 @@ void VM_Version::initialize_cpu_information(void) {
   snprintf(_cpu_desc, CPU_DETAILED_DESC_BUF_SIZE, "RISCV64 %s", features_string());
   _initialized = true;
 }
+
+bool VM_Version::is_intrinsic_supported(vmIntrinsicID id) {
+  assert(id != vmIntrinsics::_none, "must be a VM intrinsic");
+  switch (id) {
+  case vmIntrinsics::_floatToFloat16:
+  case vmIntrinsics::_float16ToFloat:
+    if (!supports_float16_to_float()) {
```

Comment:
As the `supports_float16_to_float` function name doesn't reflect the `_floatToFloat16` case, I suggest to directly inline the code here: `return UseZfh || UseZfhmin;`

Reply:
Seems to me it's better to keep it in a method, as it's used in several places. I'll change it to supports_float16_float_conversion for naming improvement.